File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.cpp
using namespace std;

long long solution(int price, int money, int count)
{
    long long answer = -1;
    
    long long now_price = 0;
    for (int i=1; i<=count; i++) {
        now_price += (price * i);
    }
    
    if (now_price <= money) answer = 0;
    else answer = now_price - money;

    return answer;
}

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js
function solution(price, money, count) {
    var answer = -1;
    
    let total = 0;
    
    for (let i=1; i<=count; i++) {
        total += (price * i);
    }
    
    if (total >= money) return (total - money);
    else return 0;
}